packages/process/foobar

```diff
@@ -0,0 +1 @@
+the quick brown fox jumps over the lazy dog
```

Comment:
Especially with the current paradigm of having package tests in the same package, it would be nice if we had a way to test this package without random test fixtures hanging out in the standard library files.